questionnaires/eq5d5l/eq5d5l_armt.json

```diff
@@ -0,0 +1,212 @@
+[
+    {
+        "field_name": "eq5d5l_1",
+        "form_name": "eq5d5l",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "Please select the ONE option that best describes your health TODAY. Mobility:",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "I have no problems in walking about"
+            },
+            {
+                "code": "1",
+                "label": "I have slight problems in walking about"
+            },
+            {
+                "code": "2",
+                "label": "I have moderate problems in walking about"
+            },
+            {
+                "code": "3",
+                "label": "I have severe problems in walking about"
+            },
+            {
+                "code": "4",
+                "label": "I am unable to walk about"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "mobility_2",
+        "form_name": "eq5d5l",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "Please select the ONE option that best describes your health TODAY. Self-care:",
```

Comment:
can `Please select the ONE option that best describes your health TODAY.` go in the section header?